csharp/downgrades/a696c8bae067f69ab3208e98ce35f4fdf7efb68b/upgrade.properties

```diff
@@ -0,0 +1,2 @@
+description: Add externalData relation
+compatibility: full
```

Comment:
should we add a `delete:` line, or is it better to just leave the table if it exists?

Reply:
That was an oversight on my part. I'll add a `delete` line.